lib/state/ui/reducer.ts

```diff
@@ -90,9 +90,20 @@ const dialogs: A.Reducer<T.DialogType[]> = (state = [], action) => {
   switch (action.type) {
     case 'CLOSE_DIALOG':
       return state.slice(0, -1);
-
+    case 'TRASH_TAG':
+      return state.filter(
+        (element) => element.type !== 'TRASH-TAG-CONFIRMATION'
+      );
```

Comment:
do you have `prettier` and the git hooks setup? I thought it would have added a newline between the `case` lines

Reply:
I believe I have it all set up properly, but it doesn't add a newline between the `case` lines. 
It looks like there is a mix of styles used after a quick search where in some places there are spaces but not everywhere.

Reply:
interesting. thanks for checking